src/App.js

```diff
@@ -3,11 +3,13 @@ import { Container } from "react-bootstrap";
 import "./App.css";
 import Footer from "./components/Footer/Footer";
 import "bootstrap/dist/css/bootstrap.min.css";
+import About from "./components/About/About";
 
 function App() {
   return (
     <>
       <Container>
+        <About />
```

Comment:
I think we need to add About as a Router in index, if it stays like this it will be rendered every time